server_admin/topics/realms/proc-creating-a-realm.adoc

```diff
@@ -13,10 +13,15 @@ realm and only be able to interact with customer-facing apps.
 
 . Point to the top of the left pane.
 
+ifeval::[{project_product}==true]
 . Click *Add Realm*.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Create Realm*.
+endif::[]
 +
-.Add realm menu
-image:{project_images}/add-realm-menu.png[Add realm menu]
+.Create realm menu
+image:{project_images}/create-realm-menu.png[Create realm menu]
```

Comment:
This image applies to the new Admin Console, but the version under rhsso-images is showing the same image. It should be unchanged for rhsso-images.

Reply:
Updated this, but please double-check if this is correct now. I may have misunderstood the directions.